libparsec/crates/protocol/tests/v3/block_create.rs

```diff
@@ -0,0 +1,97 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+// `allow-unwrap-in-test` don't behave as expected, see:
+// https://github.com/rust-lang/rust-clippy/issues/11119
+#![allow(clippy::unwrap_used)]
+
+use hex_literal::hex;
+
+use libparsec_protocol::authenticated_cmds::v3 as authenticated_cmds;
```

Comment:
should use `super::authenticated_cmds` !!! otherwise we cannot reuse this code for testing with api v4